indra/assemblers/pybel_assembler.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from builtins import dict, str
+from indra.statements import *
+from indra.databases import hgnc_client
+import logging
+import pybel
+import pybel.constants as pc
+from pybel.parser.modifiers import PmodParser
+
+# Python 2
+try:
+    basestring
+# Python 3
+except:
+    basestring = str
+
+logger = logging.getLogger('pybel_assembler')
+
+class PybelAssembler(object):
+
+    def __init__(self, stmts=None):
+        if stmts is None:
+            self.statements = []
+        else:
+            self.statements = stmts
+
+        self.model = None
+
+
+    def make_model(self, **kwargs):
+        self.model = pybel.BELGraph(**kwargs)
+        for stmt in self.statements:
+            # Convert phosphorylation statements into
+            # kin(p(agent)) => pmod(p(agent))
+            if isinstance(stmt, Phosphorylation):
```

Comment:
@cthoyt This block deals only with Phosphorylations--the idea would be to specify a canonical approach to representing each of the INDRA Statements as BEL/PyBEL, preserving all evidence text, context information, etc.